Automattic/wp-calypso
# Calypso Build: add more dependencies that need to be transpiled

#### Changes proposed in this Pull Request

The Jetpack plugin relies on Calypso Build, and we've recently ran into some issues with some dependencies:

- Query String (https://github.com/sindresorhus/query-string/issues/189)
- Split on First (https://github.com/sindresorhus/split-on-first/issues/6)
- Strict URI Encode (https://github.com/kevva/strict-uri-encode/issues/6)

Those dependencies now use es6, and consequently need to be transpiled before they can be used.

Since those are fairly general modules, I think it would make sense to patch them in Calypso Build right away; other Calypso Build users may rely on the as well, directly or indirectly.

#### Testing instructions

1. Start with [a local installation of Jetpack](https://github.com/Automattic/jetpack/blob/master/docker/README.md)
2. Run `yarn build` to build the dependencies
3. Edit `node_modules/@automattic/calypso-build/webpack/util.js` with the changes from this PR.
4. Run `yarn build-extensions`
5. Visit your site's block editor in IE11; Jetpack blocks should be available.
